scss/civicrm/common/_crm-footer.scss

```diff
@@ -0,0 +1,13 @@
+.crm-footer {
+  a,
+  a:link {
+    color: darken($brand-primary, $crm-darken-percentage);
+  }
+  .crm-error {
+    border-radius: 2px;
+  }
+}
+
+#civicrm-footer {
```

Comment:
why are we using both `.crm-footer` and `#civicrm-footer`? What's the difference? Can't we use just one?

Reply:
This border style is done with ID in core civicrm so, we are handling it by doing it with ID rather than using !important.

Reply:
then can't you use `#civicrm-footer` for all the styling, it's confusing that we are using two selectors with similar names

Reply:
Ok I will do that way. It can be done.